### DISCUSSION TITLE
mySync v1.1.0

### DISCUSSION THREAD

**Discussion by Czuz** [Czuz/mySync-for-myCloud-OS5#10] [Announcements]
# mySync for myCloud OS5 v1.1.0

Packages generated including [rclone-v1.60.0-linux-amd64](https://github.com/rclone/rclone/releases/download/v1.60.0/rclone-v1.60.0-linux-amd64.zip) binary.

## Note:
mySync has been tested only on selected NAS models hence consider packages marked below as "rc1" as a pre-release:

* MyCloudEX2Ultra_mySync_1.1.0-rc1.bin
* WDCloud_mySync_1.1.0-rc1.bin
* WDMyCloudEX4100_mySync_1.1.0-rc1.bin
* WDMyCloudMirror_mySync_1.1.0-rc1.bin
* WDMyCloud_mySync_1.1.0-rc1.bin

<hr /><em>This discussion was created from the release <a href='https://github.com/Czuz/mySync-for-myCloud-OS5/releases/tag/v1.1.0'>mySync v1.1.0</a>.</em>

**Comment by ilbloggazzo** [comment]
i still try to install on MY CLOUD 3TB White box single bay, is the frst wd but have os5, not install

**Reply by Czuz** [reply]
Hi, as I understand, you have this device:

https://www.westerndigital.com/products/cloud-storage/wd-my-cloud-home?sku=WDBVXC0030HWT-NESN

Am I correct?

**Comment by ilbloggazzo** [comment]
No, this https://global.discourse-cdn.com/wd/original/3X/2/4/24c14350a7179ac171c76d8b7bdffa209cba41ba.png



Inviato dal mio Galaxy



-------- Messaggio originale --------
Da: Czuz ***@***.***>
Data: 17/05/26 19:09 (GMT+01:00)
A: Czuz/mySync-for-myCloud-OS5 ***@***.***>
Cc: ilbloggazzo ***@***.***>, Comment ***@***.***>
Oggetto: Re: [Czuz/mySync-for-myCloud-OS5] mySync v1.1.0 (Discussion #10)


Hi, as I understand, you have this device:

https://www.westerndigital.com/products/cloud-storage/wd-my-cloud-home?sku=WDBVXC0030HWT-NESN

Am I correct?

—
Reply to this email directly, view it on GitHub<https://github.com/Czuz/mySync-for-myCloud-OS5/discussions/10#discussioncomment-16951737>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AZBIFOAZXVJLVVLVQ37YS6343HW5FAVCNFSM6AAAAACZBQJMJGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTMOJVGE3TGNY>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by Czuz** [reply]
Please confirm that you have been installing package for WDMyCloud?


Does the installation fail, or does it install successfully but not work afterward? is there any error message?

**Reply by ilbloggazzo** [reply]
Not install, if I cose a wrong version told me not support. If I install version for model i think work told me "Impossibile caricare il pacchetto dell'app per l'installazione. Verificare la connessione di rete e riprovare."

What you think for solve? thanks

**Reply by ilbloggazzo** [reply]
https://www.ebay.it/itm/397969573030?chn=ps&norover=1&mkevt=1&mkrid=724-128315-5854-1&mkcid=2&mkscid=101&itemid=397969573030&targetid=2313342991806&device=c&mktype=pla&googleloc=9201087&poi=9209998&campaignid=23210672393&mkgroupid=188737906318&rlsatarget=pla-2313342991806&abcId=10498165&merchantid=5543007720&gad_source=1&gad_campaignid=23210672393&gbraid=0AAAABBwAPiZsas8DxNrlYVb1B3ujc4O7o&gclid=CjwKCAjwrNrQBhBjEiwAoR4VOzVpsd_bUjKMjPq6InYyxRftvwLVYavnxPJrP4xH83tX7CLZQYLR4RoCC1UQAvD_BwE

**Reply by Czuz** [reply]
> Not install, if I cose a wrong version told me not support. If I install version for model i think work told me "Impossibile caricare il pacchetto dell'app per l'installazione. Verificare la connessione di rete e riprovare."
> 
> 
> 
> What you think for solve? thanks

Hi. 

I was looking at this but don't have a solution yet.

I don't have this device to test it myself and the documentation has not provided any clues so far.

Kind regards,
Czuz

**Reply by ilbloggazzo** [reply]
Write me in DM if you will i test some solutions
